generators/typescript/utils/commons/src/scripts/ScriptsManager.ts

```diff
@@ -0,0 +1,15 @@
+import fs from "fs/promises";
+import path from "path";
+
+import { AbsoluteFilePath } from "@fern-api/fs-utils";
+
+import { renameToEsmFilesFile } from "./RenameToEsmFilesFile";
+
+export class ScriptsManager {
+    public async copyScripts({ pathToRoot }: { pathToRoot: AbsoluteFilePath }): Promise<void> {
+        // make sure the scripts directory exists
+        const scriptsDir = path.join(pathToRoot, "scripts");
```

Comment:
Should we use the new `SCRIPTS_DIRECTORY_NAME` constant here? If it isn't natural to import from that package, it's ok to leave this as-is.